src/app/pages/storage/disks/disk-form/disk-form.component.ts

```diff
@@ -235,4 +229,15 @@ export class DiskFormComponent {
     value.difference = value.difference === '' ? null : value.difference;
     value.informational = value.informational === '' ? null : value.informational;
   }
+
+  inIt(pk) {
+    this.title = 'Edit Pool Disk';
```

Comment:
Bring this  title in from the helptext file where it should be wrapped in a translation marker